### PR TITLE
Remove term "Experiment" from all user facing strings

### PR DESCRIPTION
**Pull request by bhufmann**
Related to #194

The term Experiment is now removed from the UI (Trace Explorer) and
README in favor of the term trace, where a trace consists of one or more
trace files.

Signed-off-by: Bernd Hufmann <Bernd.Hufmann@ericsson.com>

### CODE REVIEW

**Review by mirsky-work — COMMENTED** [review]
LGTM, with a minor comment.

File: viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx [hunk 1]
```diff
@@ -43,9 +43,9 @@ export class TraceExplorerWidget extends ReactWidget {
     @inject(TraceViewerContribution)
     protected readonly traceViewerContribution!: TraceViewerContribution;
 
-    private OPENED_TRACE_TITLE = 'Opened experiments';
+    private OPENED_TRACE_TITLE = 'Open Traces';
```
Comment (tahini) [inline viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx:46]: If you're going to update this patch for the plural of analyses, maybe this should be "Opened Traces", or just "Traces"?

**Review by tahini — APPROVED** [review]
It's great to remote the "experiments" from everywhere! I have a simple comment on a term though

File: viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx [hunk 2]
```diff
@@ -43,9 +43,9 @@ export class TraceExplorerWidget extends ReactWidget {
...
     // private FILE_NAVIGATOR_TITLE: string = 'File navigator';
-    private ANALYSIS_TITLE = 'Available analysis';
+    private ANALYSIS_TITLE = 'Available Analysis';
```
[the remark this answers is not in the document]
Comment (bhufmann) [reply viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx:48]: Yes it should be plural. Will update at the same time.

File: viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx (hunk 1 above)
Comment (bhufmann) [reply viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx:46]: Ok, I put it back to Opened (even if I have to redo the screenshot :-))

Comment (bhufmann) [reply viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx:46]: Screenshot said already "Opened Traces"

File: README.md [hunk 3]
```diff
@@ -102,12 +100,12 @@ To open the **Trace Explorer**, select menu **View** from the top-level menu and
 
 Select the **Trace Explorer** icon to switch to the trace explorer.
 
-![Trace Explorer](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-trace-explorer-0.0.1.png)
+![Trace Explorer](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-trace-explorer-0.0.2.png)
 
-Now, you will see 3 sections: **Opened experiments**, **Available analysis** and **Time Graph Tooltip**. Select the experiment you opened and the available analysis will be populated. Now you can select different analysis and the graphs will be added into a single container view for that experiment.
+Now, you will see 3 sections: **Open Traces**, **Available Analysis** and **Time Graph Tooltip**. Select the trace that you opened and the available analysis will be populated. Now you can select different analysis and the graphs will be added into a single container view for that trace.
```
Comment (tahini) [inline README.md:105]: Update here too to Opened traces

Comment (bhufmann) [reply README.md:105]: I missed this incl. "Analyses"

**Review by tahini — APPROVED** [review]
Great, thanks!

**Comment by ebugden** [comment]
Thank you for this @bhufmann! Quick change and big impact on UX!

**Pull request merged**